src/App.js

```diff
@@ -13,6 +15,8 @@ function App() {
         <Route path='/products' element={<Products/>}/>
         <Route path='/cart' element={<Cart/>}/>
         <Route path='/login' element={<Login/>}/>
+        <Route path='/signup' element={<SignUp/>}/>
+        <Route path='/wishlist' element={<Wishlist/>}/>
```

Comment:
add page not found route so that if user type wrong url then show the page not found

Reply:
ok got it